controls/SV-204488.rb

```diff
@@ -5,39 +5,41 @@
     representing special access modes is typically ignored or required to be "0". This requirement applies to the
     globally configured system defaults and the local interactive user defaults for each account on the system.'
-  desc 'rationale', ''
-  desc 'check', 'Verify that the default umask for all local interactive users is "077".
-    Identify the locations of all local interactive user home directories by looking at the "/etc/passwd" file.
-    Check all local interactive user initialization files for interactive users with the following command:
-    Note: The example is for a system that is configured to create users home directories in the "/home" directory.
-    # grep -i umask /home/*/.*
-    If any local interactive user initialization files are found to have a umask statement that has a value less
-    restrictive than "077", this is a finding.'
+  desc 'check', %q(Verify that the default umask for all local interactive users is "077".
+
+Identify the locations of all local interactive user home directories by looking at the "/etc/passwd" file.
+
+Check all local interactive user initialization files for interactive users with the following command:
+
+Note: The example is for a system that is configured to create users home directories in the "/home" directory.
+
+$ sudo grep -ir ^umask /home | grep -v '.bash_history'
+
+If any local interactive user initialization files are found to have a umask statement that has a value less restrictive than "077", this is a finding.)
   desc 'fix', %q(Remove the umask statement from all local interactive user's initialization files.
     If the account is for an application, the requirement for a umask less restrictive than "077" can be documented with
     the Information System Security Officer, but the user agreement for access to the account must specify that the
     local interactive user must log on to their account first and then switch the user to the application account with
     the correct option to gain the account's environment variables.)
   impact 0.5
-  tag 'legacy': ['V-72049', 'SV-86673']
-  tag 'severity': 'medium'
-  tag 'gtitle': 'SRG-OS-000480-GPOS-00227'
-  tag 'gid': 'V-204488'
-  tag 'rid': 'SV-204488r603261_rule'
-  tag 'stig_id': 'RHEL-07-021040'
-  tag 'fix_id': 'F-4612r88657_fix'
-  tag 'cci': ['CCI-000318', 'CCI-000368', 'CCI-001812', 'CCI-001813',
-              'CCI-001814']
-  tag nist: ['CM-3 f', 'CM-6 c', 'CM-11 (2)', 'CM-5 (1)', 'CM-5 (1)']
-  tag subsystems: ["init_files","home_dirs"]
+  tag legacy: ['V-72049', 'SV-86673']
+  tag severity: 'medium'
+  tag gtitle: 'SRG-OS-000480-GPOS-00227'
+  tag gid: 'V-204488'
+  tag rid: 'SV-204488r861006_rule'
+  tag stig_id: 'RHEL-07-021040'
+  tag fix_id: 'F-4612r88657_fix'
+  tag cci: ['CCI-000318', 'CCI-000368', 'CCI-001812', 'CCI-001813', 'CCI-001814']
+  tag nist: ['CM-3 f', 'CM-6 c', 'CM-11 (2)', 'CM-5 (1)', 'CM-5 (1) (a)']
+  tag subsystems: ['init_files', 'home_dirs']
   tag 'host'
 
```

Comment:
BUG: Introduced profile error while trying to do DELTA update:
![image](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/assets/34140975/9a536850-c974-4452-8141-27687dad06bb)

Reply:
I changed this control, but have not confirmed if it fixes the issue for you. Feel free to test when you have a chance.